INSTALL.md

```diff
@@ -7,58 +7,46 @@ that you can download instantly and get started with fast](https://www.mapbox.co
 Still with us? These are the instructions you'll need to build Mapbox GL Native
 from source on a variety of platforms and set up a development environment.
 
-Your journey will start with getting the source code, then installing the
-dependencies, and then setting up a development environment, which varies
-depending on your operating system and what platform you want to develop for.
+Your journey will start with installing dependencies, then getting the source code, and
+then setting up a development environment, which varies depending on your
+operating system and what platform you want to develop for.
 
-## 1: Getting the source
+## 1: Installing dependencies
 
-Clone the git repository:
+### macOS
 
-    git clone https://github.com/mapbox/mapbox-gl-native.git
-    cd mapbox-gl-native
+ 1. Install [Xcode](https://developer.apple.com/xcode/)
+ 2. Launch Xcode and install any updates
+ 3. Install [Homebrew](http://brew.sh)
+ 4. Install [Node.js](https://nodejs.org/), [ruby](https://www.ruby-lang.org/), [CMake](https://cmake.org/), and [ccache](https://ccache.samba.org) (`brew install nodejs ruby cmake ccache`)
+ 5. Install [xcpretty](https://github.com/supermarin/xcpretty) (`gem install xcpretty`)
```

Comment:
We could mark this as an optional step, and remove `brew install ruby` from the list and include in here. `xcpretty` is only used for prettifying the `xcodebuild` output when building on the command line; however most developers will never do that.

Reply:
I am not sure that the typical reader of this documentation is prepared to make a decision about whether or not they plan to build on the command line. 

The compromise used by the [macOS installation flow](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/macos/INSTALL.md#building-the-sdk-from-source) may be best here: `[sudo] gem install xcpretty`